wdl-format/tests/format/seaseq-case/source.formatted.wdl

```diff
@@ -0,0 +1,939 @@
+        call util.mergehtml { input:
+            htmlfiles = indv_summarystats.xhtml,
+            txtfiles = indv_summarystats.textfile,
+            default_location = "SAMPLE",
+            outputfile = "AllMapped_" + length(fastqfiles) + "_seaseq-summary-stats.html",
+        }
+
+        call samtools.mergebam { input:
+            bamfiles = indv_mapping.sorted_bam,
+            metricsfiles = indv_bfs.metrics_out,
+            default_location = if defined(results_name) then results_name + "/BAM_files" else "AllMerge_" + length(indv_mapping.sorted_bam) + "_mapped" + "/BAM_files",
+            outputfile = if defined(results_name) then results_name + ".sorted.bam" else "AllMerge_" + length(fastqfiles) + "_mapped.sorted.bam",
+        }
+
+        call fastqc.fastqc as mergebamfqc { input:
+            inputfile = mergebam.mergebam,
+            default_location = sub(basename(mergebam.mergebam), ".sorted.b.*$", "") + "/QC/FastQC",
+        }
+
+        call samtools.indexstats as mergeindexstats { input:
+            bamfile = mergebam.mergebam,
+            default_location = sub(basename(mergebam.mergebam), ".sorted.b.*$", "") + "/BAM_files",
+        }
+
+        if (defined(blacklist)) {
+            # remove blacklist regions
+            String string_blacklist = ""  #buffer to allow for blacklist optionality
+            File blacklist_file = select_first([
+                blacklist,
+                string_blacklist,
+            ])
+            call bedtools.intersect as merge_rmblklist { input:
+                fileA = mergebam.mergebam,
+                fileB = blacklist_file,
+                default_location = sub(basename(mergebam.mergebam), ".sorted.b.*$", "") + "/BAM_files",
+                nooverlap = true,
+            }
+            call samtools.indexstats as merge_bklist { input:
+                bamfile = merge_rmblklist.intersect_out,
+                default_location = sub(basename(mergebam.mergebam), ".sorted.b.*$", "") + "/BAM_files",
+            }
+        }  # end if blacklist provided
+
+        File mergebam_afterbklist = select_first([
+            merge_rmblklist.intersect_out,
+            mergebam.mergebam,
+        ])
+
+        call samtools.markdup as merge_markdup { input:
+            bamfile = mergebam_afterbklist,
+            default_location = sub(basename(mergebam_afterbklist), ".sorted.b.*$", "") + "/BAM_files",
+        }
+
+        call samtools.indexstats as merge_mkdup { input:
+            bamfile = merge_markdup.mkdupbam,
+            default_location = sub(basename(mergebam_afterbklist), ".sorted.b.*$", "") + "/BAM_files",
+        }
+    }  # end if length(fastqfiles) > 1: multi_fastq
+
+    ### ---------------------------------------- ###
+    ### ------------ S E C T I O N 2 ----------- ###
+    ### -- B: analysis if one FASTQ provided --- ###
+    ### ---------------------------------------- ###
+
+    # if only one fastqfile is provided
+    if (one_fastq) {
```

Comment:
Personally, I don't add spacing in parenthesized expressions, but we might eventually want a configuration that would turn this into `if ( one_fastq ) {`.

Reply:
This would be an example of a opt-in type rule. Though you could see it as opt-out from the non-spaced version.